articles/getting-started-with-matlab/README.md

```diff
@@ -0,0 +1,207 @@
+##### Column vector
+You need to use semicolons between values. For example:
+```matlab
+a = [1;2;3;4]
+```
+And the output will be:
+```matlab
+a =
+
+     1
+     2
+     3
+     4
+```
+If you want to output a specific range of elements, you have to reference the vector with a colon. For example:
+```matlab
+values=[6 513 21 9 23 1 0];
+values(2:5)
+```
+The result will be:
+```matlab
+ans =
+
+   513    21     9    23
+```
+And if you want to list all of the vector elements, you can use `values(:)`.
+
+#### Matrix
+The matrix is a two-dimensional array of numbers.
+
+To create a matrix, you have to enter the elements in each row with space or colon between them. Then you have to use semicolons to delimit the rows as we saw above. For example:
+```matlab
+values=[1 2 3 4;5 6 7 8; 9 10 11 12]
+```
+The result:
+```matlab
+values =
+
+     1     2     3     4
+     5     6     7     8
+     9    10    11    12
+```
+Now let's learn how to create a smaller matrix from our existing one. For example, if you execute `values(:, 1:2)` the result will be:
+```matlab
+ans =
+
+     1     2
+     5     6
+     9    10
+```
+You can specify the rows you want before the comma, in our case, it is every row in that matrix. Also, you can specify the columns you want after the comma, in the code above it is the first and the second rows.
+
+Now go and execute `values(1:3, 2:4)` yourself. What will be the result?
+#### Zeros, Ones and Eye
+When you use `zeros` which is a built-in function, it makes every value equal to zero. For example, if you write `a = zeros(4:5)` then the result will be four rows and five columns of zeros:
+```matlab
+a =
+
+     0     0     0     0     0
+     0     0     0     0     0
+     0     0     0     0     0
+     0     0     0     0     0
+```
+`Ones` use the same method above, it makes every value equal to one. Try `b = ones(5:6)` and see the result yourself.
+
+`Eye` is a built-in function as well, it makes every value on the main diagonal equal to one and zeros elsewhere. Example:
+```matlab
+c = eye(4)
+```
+The result is:
+```matlab
+c =
+
+     1     0     0     0
+     0     1     0     0
+     0     0     1     0
+     0     0     0     1
+```
+#### Linspace 
+Linspace function generates linearly spaced vectors. The difference between colon operator and linspace is that the second one gives us control over the points. For example, if you write `linspace(1,10,10)` then the result will be:
+```matlab
+ans =
+
+     1     2     3     4     5     6     7     8     9    10
+```
+But if you change the last value to 8 instead of 10 then the result will be:
+```matlab
+ans =
+
+    1.0000    2.2857    3.5714    4.8571    6.1429    7.4286    8.7143   10.0000
+```
+
+#### M-files
+Entering commands at MATLAB prompt is not practical when the problems will be complicated. The solution will be using M-files which is user-defined commands that normally have input & output and the script will be executed in sequence when it is called.
```

Comment:
Using the MATLAB interpreter is not a wise choice for implementing and storing MATLAB code. Therefore, the code is stored in `.m` files. 

The second sentence is not clear. Change it.